english/untrustworthy/untrustworthy.md

```diff
@@ -1,4 +1,6 @@
 "If you want something done right, you have to do it yourself."
 
 Don't trust other people to do important things for you.
-You have to do things yourself to control the quality of the results.
\ No newline at end of file
+You have to do things yourself to control the quality of the results.
+
+However you ask yourself, "do you have the time?". But you already know the answer. It's time to move on, you may not get the credit you deserve, but at least you don't have to worry about someone screwing it up.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.